web/src/app/users/UserContactMethodCreateDialog.js

```diff
@@ -43,18 +43,22 @@ export default function UserContactMethodCreateDialog(props) {
 
   const { loading, error } = createCMStatus
   const fieldErrs = fieldErrors(error)
+  var title = 'Create New Contact Method'
```

Comment:
How about:

```js
const {title = 'Create New Contact Method', subtitle} = props
```

Then we can remove the ternary expressions inside the return for both.